README.md

```diff
@@ -1 +1,34 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. The only optional section is FAQ. Please remove the template logo. Optionally, you can replace with your own logo if you hsve one. This is crucial to have a professional and instructive repository. 👌

Reply:
Could not change branch name because it will close this pull request.